src/baklava/options/Vector.vue

```diff
@@ -1,11 +1,14 @@
 <template>
   <div class="d-sm-flex">
     <div class="ml-1">{{ name }}</div>
-    <IntegerInc :index=index
-                :key="index"
-                :value=val
-                @value-change="updateValue"
-                v-for="(val, index) in value"></IntegerInc>
+    <div class="d-sm-flex" v-for="(val, index) in value" :key="index">
+      <span v-if="index > 0">,</span>
```

Comment:
Why do you need the `v-if` directive here? Is the index not always positve?

Reply:
so if it's the first one I don't want to prepend a comma. Comma is only rendered if the index is not the first (i.e. 0)

Reply:
Completely missed that, thanks for the explanation